app/services/invoices/check_transition_to_finalized_service.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module Invoices
+  class CheckTransitionToFinalizedService < BaseService
```

Comment:
i think CheckInvoiceFinalStatusService explains better what this does what do you think?

Reply:
Hey!
sorry, after our discussion I still have some doubts  if instead of check it should be `Find` or `Assign`, or `Set`, or `Transition` (`TransitionInvoiceToFinalStatus`, wdyt?)... because `Check` sounds as if it should return true or false...